File: tms-webapi-master/TMS.Common/ViewModels/FilterReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TMS.Common.ViewModels
{
    public class FilterReport
    {
        public string[] ListUserID { set; get; }
        public string StartDate { set; get; }
        public string EndDate { set; get; }
    }
}

File: tms-angular2-master/src/app/core/models/filterReport.ts
export class FilterReport {
    ListUserID: string [];
    StartDate:string;
    EndDate:string;

    constructor(ListUserID: string [],StartDate:string,EndDate:string){
        this.ListUserID = ListUserID;
        this.StartDate=StartDate;
        this.EndDate=EndDate;
    }
}